### PR TITLE
Adding [Reindent Selected Lines] command.

### PR DESCRIPTION
**Pull request by malt03**
## Overview
This PR will add a command named 'Reindent Selected Lines'.
#19847
https://github.com/Microsoft/vscode/issues/28658#issuecomment-314560204

## Screenshot
reindentselectedlines

## Note
- Use #35594 as a reference.

### CODE REVIEW

**Review by nzec — APPROVED** [review]
LGTM!

**Comment by malt03** [comment]
@roblourens @rebornix What can I do to merge this pull request?

File: src/vs/editor/contrib/indentation/indentation.ts [hunk 1]
```diff
@@ -327,6 +327,48 @@ export class ReindentLinesAction extends EditorAction {
...
+			} else {
+				startLineNumber--;
+			}
+			let editOperations = getReindentEditOperations(model, startLineNumber, endLineNumber) || [];
+			for (let editOp of editOperations) {
+				edits.push(editOp);
+			}
+		}
+
+		if (edits) {
```
Comment (rebornix) [inline src/vs/editor/contrib/indentation/indentation.ts]: `edits` is an array so this condition is always true, let's check its length.

**Comment by malt03** [comment]
@rebornix Thanks you a lot!!

**Comment by malt03** [comment]
@rebornix When will this be merged?

**Comment by rebornix** [comment]
@malt03 didn't find time to review my issues/prs last week. Merged it now, you can see it from Friday's Insdiers build and Stable release for May. Sorry that I didn't make it in April release.

**Pull request merged**